libs/ui-watt/src/lib/foundations/color/storybook/color.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-card>
+    <h1>Color</h1>
+</mat-card>
+
+<mat-card *ngFor="let type of colorTypes">
+    <h2>{{type.title}}</h2>
+    <p>{{type.description}}</p>
+
+    <div class="color-grid">
+        <figure *ngFor="let color of type.colors">
+            <div class="color-sample" [style.backgroundColor]="color.color" [style.color]="color.contrast">Text on {{color.name}}</div>
```

Comment:
Question: The "Text on ..." part is not on Figma. Can you align with @stephaniegitha on that? It'd be nice if we can keep these things consistent.

Reply:
Yes we need to align this as Figma doesn't include any text / contrast colors